src/main/java/securbank/dao/OTPDao.java

```diff
@@ -0,0 +1,16 @@
+package securbank.dao;
+
+import java.util.UUID;
+
+import org.joda.time.LocalDateTime;
+
+import securbank.models.Otp;
+/**
+ * @author Joel Mascarenhas
+ *
+ */
+public interface OTPDao extends BaseDao<Otp, UUID> {
+	public int create(UUID userId,Integer code,LocalDateTime createdOn);
+	public Otp getOTP(UUID userId,Integer code);
```

Comment:
Can you change these parameters to `(User user)` and skip code.
And can you change the method to latest OTP